test/beast/http/serializer.cpp

```diff
@@ -115,10 +149,20 @@ class serializer_test : public beast::unit_test::suite
         }
     }
 
+    void testBodyWriterCtor()
+    {
+        response<deprecated_body> resp{};
+        request<deprecated_body> req{};
```

Comment:
Why are you value-initializing?

Reply:
Probably a copy-paste leftover. Will fix.